product_pack/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 NaN (http://www.nan-tic.com) - Àngel Àlvarez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Product Pack",
+    "version": "13.0.1.0.0",
+    "category": "Product",
+    "summary": "This module allows you to set a product as a Pack",
+    "website": "https://github.com/OCA/product-pack",
+    "author": "NaN·tic, " "ADHOC SA, " "Tecnativa, " "Odoo Community Association (OCA)",
```

Comment:
```suggestion
    "author": "NaN·tic, ADHOC SA, Tecnativa, Odoo Community Association (OCA)",
```
I know that black does things like this but they look a bit ugly and if we fixes it like that black won't change it again